cfme/markers/composite.py

```diff
@@ -11,6 +11,7 @@ def pytest_addoption(parser):
 
 
 def pytest_collection_modifyitems(session, config, items):
+    print('DEBUG: modifyitems in composite-uncollect')
```

Comment:
Should this be a `logger` call? Or can it be removed?

Reply:
my bad, missed cleaning this one up - I'll comb for these